src/Raven.Server/Documents/Patch/JsBlittableBridge.cs

```diff
@@ -74,11 +74,11 @@ private void WriteJsonValue(object parent, bool isRoot, string propertyName, obj
                 else if (js.IsDate())
                 {
                     var jsDate = js.AsDate();
-                    if (double.IsNaN(jsDate.PrimitiveValue) ||
-                        jsDate.PrimitiveValue > MaxJsDateMs ||
-                        jsDate.PrimitiveValue < MinJsDateMs)
+                    if (double.IsNaN(jsDate.DateValue) ||
+                        jsDate.DateValue > MaxJsDateMs ||
+                        jsDate.DateValue < MinJsDateMs)
                         // not a valid Date. 'ToDateTime()' will throw
-                        throw new InvalidOperationException($"Invalid '{nameof(DateInstance)}' on property '{propertyName}'. Date value : '{jsDate.PrimitiveValue}'. " +
+                        throw new InvalidOperationException($"Invalid '{nameof(DateInstance)}' on property '{propertyName}'. Date value : '{jsDate.DateValue}'. " +
```

Comment:
What are all the referenced for "use active realms Intrinsics to access well-known object" in the deprecation warnings?

Reply:
Many objects were transferred from `Engine` to separate slot, `Realm`. JS spec has the concept of realms and it also reflects to ability of having `ShadowRealm` separated from the "regual global context". Not sure if `Realm` is the best name or should it just be `Intrinsics` for `Engine` instance. We are trying to follow the spec with naming etc.